tensorflow_addons/layers/tests/esn_test.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for Echo State recurrent Network (ESN)."""
+
+import pytest
+import numpy as np
+import tensorflow as tf
+from tensorflow_addons.layers.esn import ESN
+from tensorflow_addons.utils import test_utils
+
+
+@pytest.mark.usefixtures("maybe_run_functions_eagerly")
+@pytest.mark.parametrize("dtype", [np.float16, np.float32, np.float64])
+def test_random(dtype):
```

Comment:
NIT: this is not a test case about randomized input :smiley: Maybe we could change its name.

Reply:
Done 😅